doc/pub/week2/ipynb/week2.ipynb

```diff
@@ -1849,7 +1849,7 @@
    "metadata": {},
    "source": [
     "$$\n",
-    "H_0\\vert 10 \\rangle = \\epsilon_{10}\\vert 10 \\rangle,\n",
+    "H_0\\vert 01 \\rangle = \\epsilon_{01}\\vert 01 \\rangle,\n",
```

Comment:
These changes are strictly speaking not necessary, but I guess it makes sense to change the ordering, because of the correction above in the definition of |01> and |10>.